### PR TITLE
Removing <html> and semi-improving

### PR DESCRIPTION
**Pull request by 8JeDaYoshi8**
It will remove the html tag which is unnecesary and allows more easy channel, server and guest nick name. Also protection for guests, for registereds-only.

### CODE REVIEW

**Comment by LifeMushroom** [comment]
This is dead, and I don't see the use of this than making stuff a litttle easier. Also, the loguserid has bad grammar.

**Comment by StapleButter** [comment]
Stop trying to convince people that you can program.

**Comment by LifeMushroom** [comment]
70% of why I rejected this was because it was JeDaYoshi's. :D

**Comment by StapleButter** [comment]
Weren't you supposed to be on good terms with him?

**Comment by LifeMushroom** [comment]
Nah, not really.

**Comment by StapleButter** [comment]
I heard that a while ago from you. What went wrong?

**Comment by LifeMushroom** [comment]
We're not enemies, but we're not friends. Imo I'm annoyed by him a little. But we can discuss that somewhere else if you want.

**Comment by 8JeDaYoshi8** [comment]
Well, is ok if you don't add it.
But StapleButter, I can program >_<

**Comment by StapleButter** [comment]
Your skills in programming are equal to my skills in flying an airplane.
And I've never flown an airplane.

File: pages/irc.php [hunk 1]
```diff
@@ -1,14 +1,22 @@
 <?php
 $title = __("IRC Page");
+if(!$loguserid)
+{
+  Kill("You should be log in for use the IRC.");
+}
+else
+{
+$nick = "insert default guest nick here";
```
Comment (xfix) [inline pages/irc.php]: No. If you require logging in (for absolutely no reason) then this is completely stupid.

Comment (StapleButter) [inline pages/irc.php]: A good reason to require login would be keeping idiots like eyelyke2shit
out.

No bonus for the shitty English in the Kill() message, though.

**Comment by LifeMushroom** [comment]
No JeDa can program
http://blargpenguin.tk/

**Comment by StapleButter** [comment]
If you call that programming, it's laughable.

File: pages/irc.php (hunk 1 above)
Comment (xfix) [inline pages/irc.php]: @StapleButter: I mean, putting default nick for logged in user.

Comment (StapleButter) [inline pages/irc.php]: Oh, right.

lol

**Comment by LifeMushroom** [comment]
That was sarcasm. :p

File: pages/irc.php (hunk 1 above)
Comment (LifeMushroom) [inline pages/irc.php]: You can easily call loguser or something to make the name of the user come on the client. I did it, but I forgot.

**Pull request closed without merging**